app.js

```diff
@@ -152,6 +114,7 @@ function reverseIds() {
   const allSquares = document.querySelectorAll(".square"); // grab everything that has the class "square"
   allSquares.forEach((square, i) =>
     // for each square, override the existing squareId with its reverse (width * width - 1) - i
+    // TODO(Martin): Oh wow what is the point of this, reversing all ids? I wonder why would we do that? Quite wild.
```

Comment:
@Martinsos The reason is that for a black player, the square IDs are different than for a white player. So A1 for the white player is not the same A1 for the black player, if that makes sense. You need to essentially switch the board with every turn. At least that's how the video explained it.

Reply:
Uff, ok! Terrible solution engineering-wise and not something I would ever recommend in real code, but I guess it can be allowed for the learning purposes (since the author of the video already went with it). Although I don't see any reason to justify it, even for educational reasons.

Reply:
Lol, I'll try your version as soon as I'm done with this one. :)